Gemfile

```diff
@@ -14,3 +14,8 @@ group :debug do
   gem "pry-stack_explorer"
   gem "rb-readline"
 end
+
+# This is a required dependency for windows platforms
+platforms :mswin, :mingw, :x64_mingw do
```

Comment:
would it help to add `:windows` here? i see in other Gemfile conditionals, we've used `if RUBY_PLATFORM.match?(/mswin|mingw|windows/)` hence confirming.

Reply:
Hey @vkarve-chef, 

Using the platform :windows is throwing error while bundling. I think we cannot use that.